docs/includes/pairlists.md

```diff
@@ -164,6 +165,29 @@ If the trading range over the last 10 days is <1%, remove the pair from the whit
 !!! Tip
     This Filter can be used to automatically remove stable coin pairs, which have a very low trading range, and are therefore extremely difficult to trade with profit.
 
+#### VolatilityFilter
+
+Volatily is the degree of historical variation of a pairs over time, is is measured by the standard deviation of logarithmic daily returns.
+
+Removes pairs where the average volatility over a `lookback_days` days is below `min_volatility` and above `max_volatility`. Since this is a filter that requires additional data, the results are cached for `refresh_period`.
+
+This filter can be used to narrow down your pairs to a certain volatilty or avoid very volatile pairs. 
+
+In the below example:
+If the volatilty over the last 10 days is not in the range of 0.20-0.30, remove the pair from the whitelist. The filter is applied every 24h.
+
+```json
+"pairlists": [
+    {
+        "method": "VolatilityFilter",
+        "lookback_days": 10,
+        "min_volatilty": 0.20,
+        "max_volatilty": 0.30,
```

Comment:
You should really fix the typos in these 2 config keys ...
i was testing this and wondering "why on earth does the default config not remove some pairs ... :laughing: 

I also think the "documented" range for this should be wider ... 
Trying this on the top 300 Volume USDT pairs doesn't leave too many pairs (now maybe i tried this at a strange time...) 
similar to other sections in the docs - this will end up as copy/paste sample, and therefore it's beneficial to have "sane" defaults.